test/framework/src/main/java/org/elasticsearch/upgrades/AbstractFullClusterRestartTestCase.java

```diff
@@ -63,6 +63,10 @@ public final boolean isRunningAgainstOldCluster() {
 
     private final Version oldClusterVersion = Version.fromString(System.getProperty("tests.old_cluster_version"));
 
+    public final boolean isRunningAgainstAncientCluster() {
+        return oldClusterVersion.before(Version.V_7_0_0);
+    }
```

Comment:
I think this method should check `runningAgainstOldCluster` (if it's really about which cluster we're running against) or be called something more like `isOldClusterAncient` (if it's just check the version of the old cluster).

Otherwise the name implies that the behaviour is like `isRunningAgainstOldCluster`,  but it is not.

Reply:
Oops, good catch @tvernum.

Reply:
The reason this works most of the time is because most of the invocations to it are already guarded by `if (isRunningAgainstOldVersion())`.